src/Nest/CommonOptions/DateFormat/DateFormat.cs

```diff
@@ -0,0 +1,162 @@
+﻿namespace Nest.CommonOptions.DateFormat
+{
+	// ReSharper disable InconsistentNaming
+	//these const reflect their output on purpose
+	public static class DateFormat
+	{
+		///<summary>A formatter for the number of milliseconds since the epoch. Note, that this timestamp is subject to the limits of a Java Long.MIN_VALUE and Long.MAX_VALUE.</summary>
+		public const string epoch_millis = "epoch_mills";
```

Comment:
kinda :) `EpochMillis(seconds)` not that bad but turning `strict_t_time` into `StrictTTIme` :/ their is litttle guessing or ambiguity in keeping the casing which is a big plus over dogmatic casing, What do you  guys reckon?